client/package.json

```diff
@@ -52,5 +52,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "inquirer": "^8.2.0",
+    "node-fetch": "^3.0.0"
   }
```

Comment:
i'm guessing these should be added to `/package.json` instead?

Reply:
Actually, now that the only place that ./data/lark/loadData.mjs is being used is inside the client container, it needs to be here and not there.  See the `services.react.volumes` and `services.django.volumes` section to see that the data is mounted to each, and used to import the fixture data depending on the type.

Legacy wise, the lark data is being imported via api calls, and the family week data is a django import.